src/applications/gqrx/receiver.cpp

```diff
@@ -194,7 +205,7 @@ void receiver::set_input_device(const std::string device)
     {
         tb->stop();
         tb->wait();
-    }
+    };
```

Comment:
Does the semicolon here have a purpose? C++ is not my strong suit but I don't think this semicolon is adding any value, right?

Reply:
Fixed in 060e0af6750fc4c83f83041fd6b99fdef6420b2b.